upf/grpc/upf.proto

```diff
@@ -0,0 +1,63 @@
+syntax = "proto3";
+
+message Fluent {
+    string name = 1;
+    string valueType = 2;
+    repeated string signature = 3;
+}
+
+message Object {
+    string name = 1;
+    string type = 2; 
+}
+
+message Expression {
+    int64 type = 1;
```

Comment:
What does this `type` represent?
At this point it is unclear to me whether we want to have `Expression` be isomorphic to an s-expresssion or if we want to go further than that.